package.json

```diff
@@ -65,15 +62,16 @@
         "@babel/plugin-syntax-dynamic-import": "^7.2.0",
         "@babel/preset-env": "^7.3.4",
         "@babel/preset-react": "^7.0.0",
-        "@material-ui/core": "^3.9.3",
-        "@material-ui/icons": "^3.0.2",
-        "@storybook/addon-actions": "^5.1.3",
+        "@material-ui/core": "^4.3.0",
+        "@material-ui/icons": "^4.2.1",
+        "@material-ui/lab": "^4.0.0-alpha.20",
```

Comment:
Can the `@material-ui` libs be moved back into `peerDependencies` now?
It seems like that's the more appropriate place for them 🤔

Reply:
@psarando Unfortunately no. If we do that, `Can't resolve '@material-ui/lab/Rating' in '/Users/sriram/cyverse-de/ui-lib/src/components/rating'` error.